float64 vx
float64 vy
float64 vz

float64 w_pitch
float64 w_roll
float64 w_yaw